doc/source/notebooks/interp_transition_guide.md

```diff
@@ -0,0 +1,249 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.0
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Interpolate transition guide
+
+This notebook contains three sets of demonstrations:
+
+- lower-level FITPACK replacements for `interp2d` for legacy bug-for-bug compatible `interp2d` replacements;
+- recommended replacements for `interp2d` for use in new code;
+- a demonstration of failure modes of 2D FITPACK-based linear interpolation and recommended replacements.
+
++++ 
+
+## 1. How to transition away from using  `interp2d`
+
+`interp2d` silently switches between interpolation on a 2D regular grid and interpolatin 2D scattered data. The switch is based on the lengths of the (raveled) `x`, `y`, and `z` arrays. In short, for regular grid use `RectBivariateSpline`; for scattered interpolation, use the `bisprep/bisplev` combo. Below we give examples of the literal point-for-point transition, which should preserve the `interp2d` results exactly.
+
++++ 
+
+### 1.1 `interp2d` on a regular grid
+
+We start from the (slightly modified) docstring example.
+
+```{code-cell} ipython3
```

Comment:
Is there a way to have this as a default, to avoid typing this for every code cell?

Reply:
Clarified this with the notebook/myst-md conversion guide.